scenarios/tracking/02_mot_challenge.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<ul><li>MOT_TRAIN_PATH and MOT_TEST_PATH need to be lower case since only user-specified variables are all upper case</li><li>Could rephrase "Since MOT17 provides ground truths in the training data" to something like "MOT17 only comes with ground truth annotations for the training set, but not for the test set. For the training set, one can upload results to evaluation server which returns the various evaluation metrics.". And also would move to the section "Evaluate on Training Set" since there is hardly any explanatory text at the moment.</li><li>The section title "pretrained model inference" is a bit misleading since it doesn't do inference. Could rename to "Model and dataset initialization"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>